packages/toolpad-app/src/server/handleDataRequest.ts

```diff
@@ -29,6 +29,18 @@ export default async (
   const apiNodeId = req.query.queryId as NodeId;
   const dom = await loadVersionedDom(appId, version);
   const api = appDom.getNode(dom, apiNodeId, 'api');
+  if (api.attributes.transformEnabled?.value) {
+    const apiResult = await execApi(
+      appId,
+      api,
+      req.query.params ? JSON.parse(req.query.params as string) : {},
+    );
+    res.json({
+      data: await evalExpression(
+        `${api.attributes.transform?.value}(${JSON.stringify(apiResult.data)})`,
```

Comment:
Isn't it already transformed inside of `execApi`?

Reply:
Yes this needn't be there - for some reason I think I found a case in the early exploration when both functions need independent transformation, but testing now and can't find such a case anymore